string nodeName
string param
string value
---
bool res
